packages/calypso-e2e/src/lib/pages/user-signup-page.ts

```diff
@@ -1,12 +1,7 @@
 import { Page } from 'playwright';
 import { getCalypsoURL } from '../../data-helper';
 import envVariables from '../../env-variables';
-import { NewUserResponse } from '../../rest-api-client';
-
-export interface NewUserDetails {
-	ID: number;
-	bearer_token: string;
-}
```

Comment:
I ended up removing the `NewUserDetails` interface in favor of returning the whole response instead and let the calling method filter out what they need.